src/uu/chmod/src/chmod.rs

```diff
@@ -195,21 +195,24 @@ impl Chmoder {
                         filename.quote()
                     );
                     if !self.quiet {
-                        return Err(USimpleError::new(
+                        show!(USimpleError::new(
                             1,
                             format!("cannot operate on dangling symlink {}", filename.quote()),
                         ));
                     }
+                    set_exit_code(1);
+                    continue;
```

Comment:
Not something to fix for this PR, but we might be able to build something like this in `uucore` maybe some verbosity level that determines whether an error is printed and always sets the exit code.

Reply:
Yeah, I think for this case, it would simplify the need for the `if self.quiet`s  and setting the exit code separately when quiet is `true`. 

I thought of two options, 
1.
```
#[macro_export]
macro_rules! show_verbose_level(
    ($err:expr, $verbose: expr) => ({
        let e = $err;
        $crate::error::set_exit_code(e.code());
        if $verbose == 1 {
            eprintln!("{}: {}", $crate::util_name(), e);
        }
    })
);
```

2. 
```

#[macro_export]
macro_rules! show_verbose_level(
    ($err:expr) => ({
        let e = $err;
        $crate::error::set_exit_code(e.code());
        eprintln!("{}: {}", $crate::util_name(), e);
    });
    ($err:expr, $verbose: expr) => ({
        let e = $err;
        $crate::error::set_exit_code(e.code());
        if $verbose == 1 {
            eprintln!("{}: {}", $crate::util_name(), e);
        }
    })
);
```

The benefit of 2) is that I guess there is no need to modify any of the existing `show!()` macros, but I guess 1) makes more sense, while we provide just 1 for verbose and 0 for non-verbose. 

This might make not too much sense, but I think thats sort of what you mean. The `macro!` name would have to show because a `show!()` without showing anything doesnt seem like the right name.